src/main/resources/assets/README.md

```diff
@@ -31,6 +31,22 @@ easy editing, set the `assets_path` flag when running Marathon:
 
         git add dist/main.js dist/main.css img/*
 
+#### Adding npm package dependencies to package.json
+
+If you want to add a new npm package to 'node_modules':
+
+1. Edit the dependencies section in package.json and add your new package.
+
+2. Install the new package
+
+	npm install [your package]
```

Comment:
`npm install [your package] --save` will install and save to dependencies in `package.json` and `--save-dev` will add it to devDependencies.